graphql/subcourse/fields.ts

```diff
@@ -318,6 +318,23 @@ export class ExtendedFieldsSubcourseResolver {
         });
     }
 
+    @FieldResolver((returns) => [Participant])
+    @Authorized(Role.ADMIN, Role.OWNER)
+    @LimitEstimated(100)
+    async pupilsOnWaitinglist(@Root() subcourse: Subcourse): Promise<Participant[]> {
+        return await prisma.pupil.findMany({
+            select: { id: true, firstname: true, lastname: true, grade: true, schooltype: true, aboutMe: true },
```

Comment:
which prop can I use to sort the waitling list according to the longest waiting students? Or will I get a sorted list from BE?

Reply:
Unfortunately this is not possible at the moment. We would have to add a timestamp to the `subcourse_participants_pupil`, but as that is a generated table from TypeORM this is a bit more complicated ... Might address this in a followup change.